gneiss/balances.py

```diff
@@ -34,9 +34,8 @@
 try:
     from ete3 import Tree, TreeStyle
     from gneiss.layouts import default_layout
-except ImportError:
-    raise ImportWarning('ete3 is not installed.  '
-                        'ETE3 style visualizations will not be available.')
+except:
```

Comment:
Here's an example of what this error looks like

<img width="1110" alt="screen shot 2017-03-07 at 7 06 10 am" src="https://cloud.githubusercontent.com/assets/4184797/23662488/a2b033ac-0304-11e7-8257-cb4ac47ec2f5.png">

Do you think this is acceptable?